pkgs/development/python-modules/cffi/default.nix

```diff
@@ -1,75 +1,46 @@
 {
-  lib,
-  stdenv,
   buildPythonPackage,
-  isPyPy,
-  fetchPypi,
-  setuptools,
-  pytestCheckHook,
+  fetchFromGitHub,
+  lib,
   libffi,
   pkg-config,
   pycparser,
+  pytestCheckHook,
+  setuptools,
+  stdenv,
 }:
 
-let
-  version = "1.17.1";
-in
-if isPyPy then
-  buildPythonPackage {
-    pname = "cffi";
-    inherit version;
-    pyproject = false;
-
-    # cffi is bundled with PyPy.
-    dontUnpack = true;
-
-    # Some dependent packages expect to have pycparser available when using cffi.
-    dependencies = [ pycparser ];
```

Comment:
Frankly, this is a bug in these packages and shouldn't live rent-free in our cffi packaging.